package.json

```diff
@@ -287,14 +287,14 @@
 		"*.scss": [
 			"wp-scripts lint-style"
 		],
-		"*.js": [
+		"*.{js,ts}": [
 			"wp-scripts format-js",
 			"wp-scripts lint-js"
```

Comment:
This will be more involved, the [linter needs a TypeScript parser](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration).